addons/gsri_opex/config.cpp

```diff
@@ -53,8 +54,8 @@ class GSRI_FREMM_Templates {
 			fullname="D-651 Normandie";
 			identifier="651";
 			name="Normandie";
-			flag="gsri_opex\images\flag_fr.paa";
-			nameplate="gsri_opex\images\normandie.paa";
+			flag="fr\gsri\opex\images\flag_fr.paa";
```

Comment:
Should we use https://github.com/team-gsri/GSRI-Gear/tree/master/addons/gsri_flags ?

Reply:
Yes, good idea ! I'll change to that !